frontend/device/kobo/powerd.lua

```diff
@@ -300,10 +300,54 @@ function KoboPowerD:isChargedHW()
     return false
 end
 
+function KoboPowerD:_postponedSetIntensityHW(end_intensity)
+    self:setIntensityHW(end_intensity)
+    self.running = false
```

Comment:
Is this actually checked anywhere? Or was it supposed to match be fl_ramp_running?